docs/tutorials/flight_controller_telemetry_tutorial.rst

```diff
@@ -145,11 +145,15 @@ the displayed RPM value based on the value reported by the module, see the :ref:
 ERPM vs. RPM
 ##############
 As of v4.5.5, ArduCopter flight controllers expect the DSHOT telemetry messages to match the `KISS ESC standard <https://www.rcgroups.com/forums/showatt.php?attachmentid=8524039&d=1450424877>`_.
-That means that it expects the module to send ERPM/100. Vertiq modules, however, directly send RPM as covered in :ref:`ESC Telemetry <manual_telemetry>`.
+That means that it expects the module to send ERPM/100. Vertiq modules by default, however, directly send RPM as covered in :ref:`ESC Telemetry <manual_telemetry>`.
 
-To convert from ERPM to RPM, the ERPM value should be divided by the pole count of the motor divided by 2. The flight controller is expecting to receive 
-ERPM/100, so when it receives telemetry it tries to calculate RPM using the following formula shown below. This calculation can also be
-seen in the `ArduCopter code for BLHeli telemetry parsing <https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_BLHeli/AP_BLHeli.cpp#L1447>`_.
+There are two methods to deal with this. The first, and recommended, option is to configure your module to transmit the ERPM/100 value directly through its telemetry responses. This can be done by changing the value of **Timer Based Protocol RPM Reporting Style** 
+in the Control Center's advanced tab:
+
+.. image:: ../_static/tutorial_images/fc_telemetry_tutorial/erpm_config.png
+    :align: center
+
+Simply set this value to **eRPM/100**, and your module will automatically transmit the KISS ESC standard's form of telemetry, and your flight controller will manage it natively as it calculates RPM by:

```

Comment:
But you will still need to set the number of poles correctly on the flight controller now, right? You need to set it to the actual number of poles instead of a wacky work around number, since it needs to know that to handle the ERPM data coming in I think. Should probably tell people that they also need to set that appropriately for their motor, and where they can find that if its something we have on the datasheets